src/actions/profile.action.js

```diff
@@ -58,7 +58,13 @@ const updateProfile = obj => {
       }
       const user = decodeToken();
       const { data } = await http.patch(`/profile/${user.id}`, obj);
-      return dispatch(updateProfileSuccess(data));
+      dispatch(updateProfileSuccess(data));
+      toast.info('You have successfully update your profile.', {
```

Comment:
please use a more descriptive error message e.g `Profile update Successful`

Reply:
I'll update that.